### PR TITLE
Remove unused fabfile tasks

### PR DESCRIPTION
**Pull request by benrudolph**
@dannyroberts did a scan on what i thought was cruft. let me know if you disagree. this file is just getting a bit unwieldy

### CODE REVIEW

**Comment by dannyroberts** [comment]
Awesome, @benrudolph, I can tell I'm gonna like this PR before even looking at it

File: fab/fabfile.py [hunk 1]
```diff
@@ -360,27 +347,6 @@ def webworkers():
...
-
-    with open(packages_file) as f:
-        packages = f.readlines()
-
-    sudo("%s %s" % (installer_command,
-                    " ".join(map(lambda x: x.strip('\n\r'), packages))), user='root')
-
-
-@task
-@roles(ROLES_ALL_SRC)
```
Comment (dannyroberts) [inline fab/fabfile.py]: Curious why you removed the @roles tag. Is this only to be run on touchforms?

Comment (dannyroberts) [inline fab/fabfile.py]: Oh i get it, I'm just a poopy face and can't read a diff

File: fab/fabfile.py [hunk 2]
```diff
@@ -391,21 +357,6 @@ def install_npm_packages():
...
-    """
-    _require_target()
-    if what_os() == 'ubuntu':
-        sudo("apt-get update", shell=False, user='root')
-        sudo("apt-get upgrade -y", shell=False, user='root')
-    else:
-        return
-
-
 @task
```
Comment (dannyroberts) [inline fab/fabfile.py]: Ok but here though. I think you've got an extra `@task` decorator

**Comment by dannyroberts** [comment]
:+1: 0 additions and 144 deletions :smile:

Looks good to me, will let you respond to my nitpick

File: fab/fabfile.py (hunk 1 above)
Comment (benrudolph) [inline fab/fabfile.py]: haha this diff confused me so much when i was reviewing it. git diff should weight contiguous lines of code more strongly

File: fab/fabfile.py (hunk 2 above)
Comment (benrudolph) [inline fab/fabfile.py]: as in you don't think `what_os` should be a task? i'm down with that. also down with thinking about removing that entire function

Comment (benrudolph) [inline fab/fabfile.py]: oh shit i see what you're saying. oops!

**Comment by snopoke** [comment]
:+1: :+1: :+1: :+1:

**Comment by benrudolph** [comment]
k comment addressed

**Pull request merged**